src/radical/pilot/agent/scheduler/flux.py

```diff
@@ -144,5 +144,49 @@ def work(self, tasks):
   #     return new_env
 
 
+    # --------------------------------------------------------------------------
+    #
+    def task_to_spec(self, task):
+
+        td   = task['description']
+        cmd  = '%s %s 1>%s 2>%s' % (td['executable'],
+                      ' '.join(td['arguments']), task['stdout'], task['stderr'])
```

Comment:
instead of `task['stdout'], task['stderr']` should be `td['stdout'], td['stderr']`, which also should be checked - e.g., `td.get('stdout') or '%s.out' % tid` (similar to what `Popen` does)

Reply:
Ack, thanks!